docs/arch/adr-18.md

```diff
@@ -0,0 +1,47 @@
+# ADR 18: Workflow routing
+
+Created: September 4, 2019
+
+## Context
+
+On PFE, there are a couple of different ways workflows are routed:
+
+- Navigating to `/classify`, the user sees either the default workflow for that project, or the workflow stored in project preferences for logged-in volunteers.
+- Navigating to `/classify?workflow=[workflow_id]` shows the workflow determined by the query parameter.
```

Comment:
If enabled for the project or if the user has the correct role: owner, collaborator, tester, or admin